beta1/integrated-build.json

```diff
@@ -93,8 +93,8 @@
         "strategy": {
           "type": "STI",
           "stiStrategy": {
-            "image": "openshift/ruby-20-centos",
-            "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
+            "image": "openshift/ruby-20-centos7",
+            "scripts": "https://raw.githubusercontent.com/openshift/sti-ruby/master/2.0/.sti/bin"
```

Comment:
@mfojtik what version of "ex generate" did you run? Did you use the beta1 code?

@soltysh We only need to keep ruby-20-centos until beta2 drops. Once beta2 drops, beta1 is no longer valid.